pom.xml

```diff
@@ -81,7 +81,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
+    <cdap.version>6.1.0-SNAPSHOT</cdap.version>
```

Comment:
plugins should avoid snapshots unless they're actually using new, unreleased functionality. This should be 6.0.0. Can you also change 2.2.0-SNAPSHOT?